portfolio/src/main/data_processing/.gitignore

```diff
@@ -0,0 +1,2 @@
+# created by virtualenv automatically
+*
```

Comment:
Weird, this doesn't also exclude all the other files that you actually do want to keep? Shouldn't this point to the actual virtual env folder?

Also: you should create your virtual env folder at the root of the repository, just so it's easy to stay consistent between multiple engineers.

Reply:
I'll do that in another PR.